src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_CollectionExpression.cs

```diff
@@ -128,7 +191,7 @@ private BoundExpression VisitArrayOrSpanCollectionExpression(BoundCollectionExpr
 
                 Debug.Assert(IsAllocatingRefStructCollectionExpression(node, collectionTypeKind, elementType.Type, _compilation));
                 arrayType = ArrayTypeSymbol.CreateSZArray(_compilation.Assembly, elementType);
-                spanConstructor = ((MethodSymbol)_compilation.GetWellKnownTypeMember(
+                spanConstructor = ((MethodSymbol)_factory.WellKnownMember(
```

Comment:
nit: `_factory.WellKnownMethod` eliminates the need to cast

Reply:
Actually, we're using `_factory.WellKnownMethod()` throughout this file, and in a number of other callsites in `LocalRewriter` where we could cast the result to `MethodSymbol`. Given that, I'll leave as-is, to reduce the churn.